src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -3,13 +3,14 @@
 
 #include "pch.h"
 #include "Appearances.h"
-#include "Appearances.g.cpp"
-#include "AxisKeyValuePair.g.cpp"
-#include "FeatureKeyValuePair.g.cpp"
 
-#include "EnumEntry.h"
 #include <LibraryResources.h>
-#include "..\WinRTUtils\inc\Utils.h"
+#include "../WinRTUtils/inc/Utils.h"
+
+#include "EnumEntry.h"
+#include "ProfileViewModel.h"
+
+#include "Appearances.g.cpp"
```

Comment:
Welcome to the doomsday diff. This was awful to write, so I'm sure it's going to be awful to read.